assessment_software/src/app/model/course-card/course-card.model.ts

```diff
@@ -0,0 +1,19 @@
+export class CourseCard {
+   
+    public courseName:string;
+    public year:Number;
+    public semester:Number;
+    public assignments:Number;
+    public submissions:Number;
+
+    constructor (courseName:string, year:Number, semester:Number, assignments:Number, submissions:Number){
+        this.courseName=courseName;
+        this.year=year;
+        this.semester=semester;
+        this.assignments=assignments;
+        this.submissions=submissions;
+    }
+    getSemesterYear(){
+        return String(this.year) + '/' + this.semester;
```

Comment:
This is amazing. The fact that you made drived fields methods shows you are understanding things pretty well. You rock.